introduction.Rmd

```diff
@@ -315,14 +315,14 @@ data %>%
   mutate(rescaled = x / sd(x))
 ```
 
-In fact, `mutate()` enforces vectorisation. Returning smaller vectors is an error unless if they have size 1. If the result of a mutate expression is a constant, it is automatically recycled to the tibble or group size. This ensures that all columns have the same length and fit within the tibble constraints of rectangular data:
+In fact, `mutate()` enforces vectorisation. Returning a smaller vector is an error unless it has size 1. If the result of a mutate expression has size 1, it is automatically recycled to the tibble or group size. This ensures that all columns have the same length and fit within the tibble constraints of rectangular data:
 
 ```{r}
 data %>%
-  mutate(constant = sd(x))
+  mutate(sigma = sd(x))
 ```
 
-In contrast to `mutate()`, the dplyr verb `summarise()` expects summary operations that return single constants:
+In contrast to `mutate()`, the dplyr verb `summarise()` expects summary operations that return a single value:
 
```

Comment:
Theme of the above 3 edits: I've emphasized "has size 1". I think "constant" is confusing because the vector (1, 1, 1) is constant, in some sense, but it's not OK because it definitely doesn't have size 1. I think if you want to use specialized vocabulary here, it would be worth explaining "scalar" and using it.